source/contracts/trading/Orders.sol

```diff
@@ -18,11 +19,8 @@ contract Orders is DelegationTarget, IOrders {
     using Order for Order.Data;
     using SafeMathUint256 for uint256;
 
-    event CancelOrder(address indexed market, address indexed sender, uint256 price, uint256 amount, bytes32 orderId, uint8 outcome, Order.TradeTypes orderType, uint256 cashRefund, uint256 sharesRefund);
     event BuyCompleteSets(address indexed sender, address indexed market, uint256 amount, uint256 numOutcomes);
     event SellCompleteSets(address indexed sender, address indexed market, uint256 amount, uint256 numOutcomes, uint256 creatorFee, uint256 reportingFee);
```

Comment:
Why weren't these moved over?  Does the UI not need them?  If not, I wonder if we can just delete them.  cc @tinybike

Reply:
Are there any other events anywhere else in the contracts that aren't part of Augur.sol and needed by the UI?

Reply:
Yes I think these can just be deleted.  IIRC all the relevant information (read: fees) is contained in the order filled logs.